lib/addons/src/types.ts

```diff
@@ -267,6 +267,15 @@ export interface BaseMeta<ComponentType> {
    * @see [Story Hierarchy](https://storybook.js.org/docs/basics/writing-stories/#story-hierarchy)
    */
   title?: string;
+  
+  /**
+   * Manually set the id of a story, which in particular is useful if you want to rename stories without breaking permalinks.
+   *
+   * Storybook will prioritize the id over the title for ID generation, if provided, and will prioritize the story.name over the export key for display.
```

Comment:
The property to set a story name is now called storyName, could you change it?
 @jonniebigodes this is also mentioned wrong in [the docs](https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalinking-to-stories)
```suggestion
   * Storybook will prioritize the id over the title for ID generation, if provided, and will prioritize the story.storyName over the export key for display.
```